tests/test_config.py

```diff
@@ -596,3 +597,106 @@ def test_load_config(self):
             "initStrategy": "auto"
         })
 
+    def test_config_to_python_with_data_objects(self):
+        vc = VitessceConfig()
+
+        class MockWrapperA(AbstractWrapper):
+            def __init__(self, name, **kwargs):
+                super().__init__(**kwargs)
+                self._repr = make_repr(locals())
+                self.name = name
+            def convert_and_save(self, dataset_uid, obj_i):
+                def get_molecules(base_url):
+                    return {
+                        "url": f"{base_url}/molecules",
+                        "type": "molecules",
+                        "fileType": "molecules.json"
+                    }
+                def get_cells(base_url):
+                    return {
+                        "url": f"{base_url}/cells",
+                        "type": "cells",
+                        "fileType": "cells.json"
+                    }
+                self.file_def_creators += [get_molecules, get_cells]
+
+        class MockWrapperB(AbstractWrapper):
+            def __init__(self, name, **kwargs):
+                super().__init__(**kwargs)
+                self._repr = make_repr(locals())
+                self.name = name
+            def convert_and_save(self, dataset_uid, obj_i):
+                def get_cell_sets(base_url):
+                    return {
+                        "url": f"{base_url}/cell-sets",
+                        "type": "cell-sets",
+                        "fileType": "cell-sets.json"
+                    }
+                self.file_def_creators += [get_cell_sets]
+
+        dataset = vc.add_dataset(name='My Object Dataset').add_object(
+            obj=MockWrapperA("Experiment A")
+        ).add_object(
+            obj=MockWrapperB("Experiment B")
+        )
+
+        vc.add_view(dataset, cm.SPATIAL, x=1, y=2, w=3, h=4, mapping="PCA").set_props(title="My spatial plot")
+
+        vc_dict = vc.to_dict(base_url="http://localhost:8000")
+        
+        classes_to_import, code_block = vc.to_python()
+        
+        reconstructed_vc = eval(code_block)
+        reconstructed_vc_dict = reconstructed_vc.to_dict(base_url="http://localhost:8000")
+
+        self.assertEqual(set(classes_to_import), {"VitessceConfig", "MockWrapperA", "MockWrapperB"})
```

Comment:
Import order could, conceivably, make a difference... Is there are reason to just check set equivalence? Downstream, what will the `from ...` be?